packages/teleterm/src/services/pty/ptyService.ts

```diff
@@ -46,7 +47,7 @@ function buildOptions(settings: RuntimeSettings, cmd: PtyCommand): PtyOptions {
   switch (cmd.kind) {
     case 'new-shell':
       return {
-        path: 'bash',
+        path: getDefaultShell(),
```

Comment:
Lets move `getDefaultShell` to the [runtimeSettings.ts](https://github.com/gravitational/webapps/blob/gzdunek/use-default-terminal-shell-and-add-context-menu/packages/teleterm/src/mainProcess/runtimeSettings.ts#L10) so then it can be used as 

```ts
switch (cmd.kind) {
    case 'new-shell':
      return {
        path: settings.defaultShell,
```

Reply:
Done